tests/ast/ast_test_utils.py

```diff
@@ -44,11 +45,13 @@ def render(ast_base64: Union[str, List[str]], unparser_jar: Optional[str]) -> st
     return res.stdout
 
 
-def generate_error_trace_info(python_text):
+def generate_error_trace_info(python_text, ex=None):
```

Comment:
nit: I think it would be more readable to have exception/a full word as the parameter name instead of ex.

Reply:
Updated